grpc/src/main/proto/model/workflowtask.proto

```diff
@@ -36,4 +36,6 @@ message WorkflowTask {
     bool rate_limited = 20;
     repeated string default_exclusive_join_task = 21;
     bool async_complete = 23;
+    string loop_condition = 24;
```

Comment:
I think this and AbstractProtoMapper are autogenerated files by gradle. Consider not checking them in.